src/core_modules/capture-core/metaDataStoreLoaders/maintenance/upkeepUserCaches.js

```diff
@@ -19,6 +18,7 @@ async function addUserCacheToHistory(
     mainStorageController: StorageController,
     userStorageController: StorageController,
 ) {
+    const { applicationStoreNames: mainStores } = getContext();
```

Comment:
I wouldnt change the name here? You can feel free to keep it if you believe is needed. To me the applicationStoreNames in this case is making sense.

Reply:
This is a potential refactor task later (mainStores used elsewhere in the app)